public/app/features/dashboard/api/publicDashboardApi.ts

```diff
@@ -29,6 +29,7 @@ const backendSrvBaseQuery =
       );
       return { data: responseData, meta };
     } catch (error) {
+      // @ts-ignore
       return requestOptions.manageError ? requestOptions.manageError(error) : { error };
```

Comment:
Do not disable the type checker. This will break Grafana in unknown ways. We cannot introduce more type holes and unreliability.

Reply:
I didn't know about that isFetchError function. Looks better now!